### ISSUE TITLE
Building up and printing the 'LCS' for LongestCommonSubsequence

### ISSUE THREAD

**Issue by RohitJain3241** [suresh-gandhi/DP#2]
We could also add the backtracking method to implement the printing of LCS.

**Comment by suresh-gandhi** [comment]
Yeah we can do that! Send me a PR. Write the code at the bottom of my file in C++(which you like). I'll merge it.
